docs/templates/readme-example-template.md

```diff
@@ -0,0 +1,55 @@
+# Title of the function example
+
```

Comment:
I think adding a short description/introduction under the title (kinda like this https://www.scaleway.com/en/docs/serverless/functions/how-to/connect-function-to-object-storage/) could be beneficial.
Maybe something like: "When calling a function from a frontend application in a browser, you may receive an error about missing CORS headers. Here's how you can allow cross-origin requests to a Python function"